lottie/src/main/java/com/airbnb/lottie/manager/ImageAssetManager.java

```diff
@@ -121,6 +121,10 @@ public void setDelegate(@Nullable ImageAssetDelegate assetDelegate) {
       Logger.warning("Unable to decode image.", e);
       return null;
     }
+    if (bitmap == null) {
+      Logger.warning("Decoded image `" + id + "` is null.");
```

Comment:
Could you add the same message on line 121?

Reply:
OK